Rdmp.Dicom/CommandExecution/ExecuteCommandCreateNewImagingDatasetSuite.cs

```diff
@@ -261,9 +263,9 @@ private void SetArgument(IArgument[] args, string property, object value)
             var arg = args.Single(a => a.Name.Equals(property));
 
             var mef = ((CatalogueRepository) arg.Repository).MEF;
-            var found = mef.GetType(value.GetType().FullName);
+            mef.GetType(value.GetType().FullName);
 
-            //if this fails, look to see if found is null (indicates that your Type is not loaded by MEF).  Look at mef.DescribeBadAssembliesIfAny() to investigate this issue
+            //if this fails, look to see if GetType returned null (indicates that your Type is not loaded by MEF).  Look at mef.DescribeBadAssembliesIfAny() to investigate this issue
```

Comment:
Can we add a null check on local variable 'found' and throw if it is null? (I assume value is never intentionally null at this point?).  Id rather repair this dodgy old code than just update it like this.